packages/grid/_modules_/grid/models/api/filterApi.ts

```diff
@@ -12,4 +13,5 @@ export interface FilterApi {
   applyFilterLinkOperator: (operator: GridLinkOperator) => void;
   onFilterModelChange: (handler: (params: GridFilterModelParams) => void) => void;
   setFilterModel: (model: FilterModel) => void;
+  getVisibleRowModels: () => GridRowModel[];
```

Comment:
I think that we should fix one problem at a time. The bug is one thing. The missing API is another.

```suggestion
```